src/handler.ts

```diff
@@ -43,7 +45,12 @@ export async function handlePullRequest (context: Context): Promise<void> {
   }
 
   if (config.addAssignees) {
-    result = await pullRequest.addAssignees(owner, repo, prNumber, reviewers)
+    const assignees: string[] = config.assignees ?
+      chooseUsers(owner, config.assignees, config.numberOfAssignees || config.numberOfReviewers)
```

Comment:
I'm not certain on the `||` usage here because I'm not certain how the `?` affects things in the interface AppConfig…